src/main/java/jenkinsci/plugins/influxdb/generators/SonarQubePointGenerator.java

```diff
@@ -253,45 +248,55 @@ protected String getResult(String url) throws IOException {
         }
     }
 
-    private String[] getSonarProjectURLFromBuildLogs(Run<?, ?> build) throws IOException {
+    private String[] getSonarProjectFromBuildReport() throws IOException {    
+        String projName = null;
         String url = null;
-        String taskid = null;
-
-        try (BufferedReader br = new BufferedReader(build.getLogReader())) {
+        String taskId = null;
+        String taskUrl = null;
+        String file = env.get("WORKSPACE", "") + SONAR_BUILD_REPORT;
```

Comment:
Add support for `SONAR_SCANNER_METADATA_FILE_PATH`. This should then be documented in SonarQube integration guidelines.
```suggestion
        String default_file = env.get("WORKSPACE", "") + SONAR_BUILD_REPORT;
        String file = env.get("SONAR_SCANNER_METADATA_FILE_PATH", default_file);
```

Reply:
Not relevant with the new changes.